src/app/account/account-sidebar/account-sidebar.component.scss

```diff
@@ -11,9 +11,9 @@
 }
 
 :host /deep/ .mat-expansion-panel-content {
-  color: rgba(0, 0, 0, 0.54)
+  color: rgba(0, 0, 0, 0.54);
 }
 
 :host /deep/ .mat-tab-link {
-  min-width: 140px !important;
+  min-width: 50% !important;
```

Comment:
Can we use flex instead of hardcoding percents?

Reply:
these elements in a flex container.

but I didn't catch how to do this without setting a width
cause we need that tabs take all space in a container